newsfragments/3014.bug.rst

```diff
@@ -0,0 +1,10 @@
+Resolve function_abi AFTER condition is initialized 
```

Comment:
This file should be caled `3014.bugfix.rst`. See project.toml for the different naming conventions.